bridge/bridge/api/static/openapi/openapi.yaml

```diff
@@ -0,0 +1,919 @@
+    payout status, errors, and fee/conversion estimates.
+    Unwrap operations are available only when the bridge is running in wrap or stake mode
+    (they are not available in swap mode).
+  version: 1.0.0
+servers:
+  - url: "../../"
+    description: Current server
+  - url: "http://localhost:5000"
+    description: Local development server
+tags:
+  - name: Bridge
+    description: Provides access to bridge configuration.
+  - name: Wrap
+    description: Endpoints related to wrapping tokens from the native chain into the wrapped chain.
+  - name: Unwrap
+    description: |
+      Endpoints related to unwrapping tokens from the wrapped chain back into the native chain.
+      Unwrap operations are supported only in wrap and stake bridge modes (please check the bridge README for more details).
+
+paths:
+  /:
+    get:
+      summary: Returns bridge configuration.
+      tags:
+        - Bridge
+      operationId: getBridgeStatus
+      responses:
+        200:
+          description: Detailed bridge configuration.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BridgeConfiguration'
+        500:
+          description: Unexpected internal error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+
+  /wrap/requests:
+    get:
+      summary: Returns wrap requests information.
+      description: |
+        Retrieves a paginated list of wrap requests detected on the native network.
+        These represent user-initiated deposits that trigger wrapping.
+      tags:
+        - Wrap
+      operationId: getWrapRequests
+      parameters:
+        - $ref: '#/components/parameters/Offset'
+        - $ref: '#/components/parameters/Limit'
+        - $ref: '#/components/parameters/Sort'
+        - $ref: '#/components/parameters/PayoutStatus'
+      responses:
+        200:
+          description: List of wrap requests recorded by the bridge.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/WrapRequest'
+        400:
+          description: Invalid request parameters (e.g., invalid pagination or filters).
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        500:
+          description: Unexpected internal error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+
+  /wrap/requests/{address}:
+    get:
+      summary: Returns wrap requests filtered by sender or destination address.
+      description: |
+        Retrieves wrap requests associated with the provided address.
+        This includes both source (sender) and destination addresses.
```

Comment:
Same comments as before: clarify if the requests are still ongoing or not, and include as many links as possible to related topics.